src/routes/+layout.svelte

```diff
@@ -26,15 +30,18 @@
     listen("new-room", (e: any) => {
       const room_id = e.payload.room_id;
       const peer_id = e.payload.peer_id;
-      console.log(`new room : ${peer_id}:${room_id}`);
+      setItem(peer_id, JSON.stringify(UserStatus.Connected));
```

Comment:
Shouldn't this be changed to the new Tauri Commands too?
Also shouldn't the ``$lib/store`` be removed? (Unless it's still used for session storage)